www/analytics/last-score-by-activity-by-user/last-score-by-activity-by-user-cut-score1.php

```diff
@@ -0,0 +1,169 @@
+<?php
+use LearnositySdk\Request\Init;
+
+$lastScoreBABUReportConfigCutScore11 = array_merge($lastScoreBABUReportConfig, ['id' => "lastscore-by-activity-by-user-cutscore1"]);
```

Comment:
Could you just use `$lastScoreBABUReportConfigCutScore1`, because our customers might check the source code to understand what to do for cutscore, and `11` on the variable name might confuse a bit.